src/main/java/io/cdap/plugin/gcp/bigquery/action/BigQueryExecute.java

```diff
@@ -247,15 +248,10 @@ public AbstractBigQueryActionConfig getConfig() {
     @Macro
     private String rowAsArguments;
 
-    public Config(String dialect, String sql, String mode) {
```

Comment:
Why did you condense these 2 constructors into one? Any particular reason?

Reply:
This was done specifically to make use of the Config build() function on L486 (these fields weren't set earlier)